src/Microsoft.DotNet.Files/ContentFiles.cs

```diff
@@ -60,6 +61,24 @@ public void StructuredCopyTo(string targetDirectory)
             RemoveAttributeFromFiles(pathMap.Values, FileAttributes.ReadOnly);
         }
 
+        public void StructuredCopyToFromIncludeEntries(string targetDirectory, IEnumerable<IncludeEntry> includeEntries)
+        {
+            foreach (var targetDir in includeEntries
+                .Select(f => Path.GetDirectoryName(f.TargetPath))
+                .Distinct()
```

Comment:
I am also not sure. I did this because the existing code has the same logic. https://github.com/dotnet/cli/pull/2493/files#diff-26aa7719c4abb7f09145d4335330f687R47